README.md

```diff
@@ -64,18 +63,30 @@ Follow these steps:
     >**NOTE:** For more details about testing, go to the [Testing Kyma](https://kyma-project.io/docs/root/kyma#details-testing-kyma) document.
 
 
-### Project structure
-
-The repository has the following structure:
-
-  ```
-  ├── .github                     # Pull request and issue templates             
-  ├── components                  # Source code of all Kyma components
-  ├── docs                        # Documentation source files
-  ├── installation                # Installation scripts     
-  ├── resources                   # Helm charts and Kubernetes resources for the Kyma installation
-  ├── tests                       # Acceptance tests
-  ├── common                      # Common Go libraries
-  ├── scripts                     # Scripts used for building
-  └── tools                       # Source code of utilities used, for installation, testing, and more
-  ```
+## Kyma users
```

Comment:
instruction from @magicmatatjahu will need info for maintainer that merges, that he needs to manually add new company here